manticore/core/cpu/disasm.py

```diff
@@ -141,6 +138,18 @@ def unimplemented(self, il):
         return (il.operation == enums.LowLevelILOperation.LLIL_UNIMPL or
                 il.operation == enums.LowLevelILOperation.LLIL_UNIMPL_MEM)
 
+    def _get_current_llil_func(self, code, pc):
+        if pc in self.llil_func_cache:
```

Comment:
You mean for cases like self-modifying code? I think one of the biggest issues right now is the fact that we maintain the il_queue for instructions like `div`, `if` etc. Also there are multiple calls to binja which should be minimized, because just the mere task of fetching one 
instruction resolves into multiple calls. Perhaps it is worth it adding a feature where we load the IL directly from a serialized object, with all the functions and indexes stored there, so that we don't do any lookups.